src/docker/client/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# Copyright 2023 Flower Labs GmbH. All Rights Reserved.
+
+ARG BASE_REPOSITORY=flwr/base
+ARG BASE_IMAGE_TAG
+FROM $BASE_REPOSITORY:$BASE_IMAGE_TAG
+
+ARG FLWR_VERSION
+RUN python -m pip install -U --no-cache-dir flwr"[rest, simulation]"==${FLWR_VERSION}
```

Comment:
@tanertopal I'm unsure, if we need the `simulation` feature

Reply:
@danieljanes @jafermarq Not sure either. Do we need "simulation"
 in the client image? I would assume no but please correct me if I am wrong.

Reply:
we don't

Reply:
I removed it